includes/class-core-tweaks.php

```
@@ -12,190 +12,196 @@
+				add_action( 'wp_enqueue_scripts', array( $this, 'disable_dashicons' ) );
+			}
 
-		if ( ! empty( $this->settings['disableEmbeds'] ) ) {
-			add_action( 'init', array( $this, 'disable_embeds' ), 9999 );
-		}
+			if ( ! empty( $this->settings['disableXMLRPC'] ) ) {
+				if ( function_exists( 'wp_xmlrpc_server_class' ) ) {
+					add_filter( 'wp_xmlrpc_server_class', '__return_empty_string' );
+				} else {
+					add_filter( 'xmlrpc_enabled', '__return_false' );
+				}
```

PHPMD (2.15.0)

[error] The method __construct uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)